lib/linguist/languages.yml

```diff
@@ -2416,6 +2416,15 @@ Gleam:
   - ".gleam"
   tm_scope: source.gleam
   language_id: 1054258749
+Glimmer:
+  type: programming
+  extensions:
+  - ".gjs"
+  - ".gts"
+  ace_mode: javascript
```

Comment:
Do we need to add gjs/gts support to the ace editor?

I wrote a parser/syntax package for codemirror 6 -- i wonder if it's similar?

Reply:
From the docs:

> A String name of the Ace Mode used for highlighting whenever
>                         a file is edited. This must match one of the filenames in https://gh.io/acemodes.
>                         Use "text" if a mode does not exist.

So a full ace mode would probably improve the web editing experience. I don't think that should block this though. We can update that field later.